packages/ppx/postinstall.js

```diff
@@ -0,0 +1,18 @@
+const { exec } = require("child_process");
+const fs = require("fs");
+
+exec("uname", (error, stdout, stderr) => {
```

Comment:
now that we are using a node script for this we can just use `process.platform` instead of uname!

Reply:
Done :-) thanks